test/test.js

```diff
@@ -237,6 +237,15 @@ describe('postcss-colors-only', function () {
             done
         );
     });
+    
```

Comment:
@olegafx This is the line causing the CI to fail.

```
test/test.js
  240:1  error  Trailing spaces not allowed  no-trailing-spaces
```

Making this line a blank line instead of 4 spaces should fix it. Do that and I'll merge. :D